subworkflows/local/annotate_structural_variants.nf

```diff
@@ -33,16 +37,62 @@ workflow ANNOTATE_STRUCTURAL_VARIANTS {
             }
             .set { ch_svdb_dbs }
 
-        SVDB_QUERY(
+        ch_sv_bedpedbs
+            .splitCsv ( header:true )
+            .multiMap { row ->
+                bedpedbs: row.filename
+                in_frqs:  row.in_freq_info_key
+                in_occs:  row.in_allele_count_info_key
+                out_frqs: row.out_freq_info_key
+                out_occs: row.out_allele_count_info_key
+            }
+            .set { ch_svdb_bedpedbs }
+
+        SVDB_QUERY_DB (
             ch_vcf,
             ch_svdb_dbs.in_occs.toList(),
             ch_svdb_dbs.in_frqs.toList(),
             ch_svdb_dbs.out_occs.toList(),
             ch_svdb_dbs.out_frqs.toList(),
-            ch_svdb_dbs.vcf_dbs.toList()
+            ch_svdb_dbs.vcf_dbs.toList(),
+            []
         )
 
-        PICARD_SORTVCF(SVDB_QUERY.out.vcf, ch_genome_fasta, ch_genome_dictionary)
+        ch_vcf
+            .join(SVDB_QUERY_DB.out.vcf, remainder: true)
+            .branch { it  ->
+                call: it[2].equals(null)
+                    return [it[0], it[1]]
+                db: !(it[2].equals(null))
```

Comment:
Is it possible to use some more descriptive branch names? I guess that the call branch is basically the original vcf channel and the db branch is the possible output from the SVDB_QUERY:DB output in the case where it was part of the run. Correct?